src/components/ebay-notice/index.marko

```diff
@@ -1,77 +1 @@
-import processHtmlAttributes from "../../common/html-attributes"
-
-static var ignoredAttributes = [
-    "type",
-    "fillWindow",
-    "status",
-    "hidden",
-    "content",
-    "title"
-];
-
-$ var type = input.type || "page";
-$ var status = input.status || "attention";
-$ var isSection = type === "section";
-$ var isWindow = type === "window";
-$ var isWindowTitle = isWindow && input.title;
-$ var isCelebration = status === "celebration";
-$ var isPage = type === "page" || isSection || isWindow;
-$ var isLightGuidance = isSection && !input.status;
-$ var isIconVisible = input.icon !== "none";
-<${isPage ? "section" : "div"}
-    ...processHtmlAttributes(input, ignoredAttributes)
-    role=(isPage && "region")
-    aria-labelledby:scoped="status"
-    aria-label=(isLightGuidance && input.a11yHeadingText)
-    class=[
-        `${type}-notice`,
-        !isLightGuidance && !isWindow && `${type}-notice--${status}`,
-        input.fillWindow && `${type}-notice--fill`,
-        input.class
-    ]>
-    <if(!isLightGuidance && (isIconVisible || isWindowTitle))>
-        <${isPage ? input.a11yHeadingTag || "h2" : "span"}
-            class=`${type}-notice__status`
-            id:scoped="status">
-            <if(isWindowTitle)>
-                <if(isIconVisible)>
-                    <ebay-icon
-                        class="window-notice__icon"
-                        name="confirmation-filled"
-                        aria-label=input.a11yHeadingText/>
-                </if>
-                <span
-                    ...processHtmlAttributes(input.title)
-                    class=[`${type}-notice__title`, input.title.class]>
-                    <${input.title}/>
-                </span>
-            </if>
-            <else>
-                <if(status === "confirmation" || status === "celebration")>
-                    <ebay-icon name="confirmation-filled" aria-label=input.a11yHeadingText/>
-                </if>
-                <else if(status === "attention")>
-                    <ebay-icon name="attention-filled" aria-label=input.a11yHeadingText/>
-                </else>
-                <else if(status === "information")>
-                    <ebay-icon name="information-filled" aria-label=input.a11yHeadingText/>
-                </else>
-            </else>
-        </>
-    </if>
-    <${isPage ? "div" : "span"}
-        ...(input.content && processHtmlAttributes(input.content))
-        class=[`${type}-notice__content`, input.content && input.content.class]>
-        <if(isCelebration && input.title)>
-            <span
-                ...processHtmlAttributes(input.title)
-                class=[`${type}-notice__title`, input.title.class]>
-                <${input.title.renderBody}/>
-            </span>
-        </if>
-        <${input.content || input.renderBody}/>
-    </>
-    <if(input.content)>
-        <${input.renderBody}/>
-    </if>
-</>
+<!-- Removed -->>
```

Comment:
This file should not be necessary.

Reply:
If i removed it, it doesn't work properly, says tag is not found.

Reply:
Can you share the full error? That would mean something bad is happening.